lib/ui/painting/canvas.h

```diff
@@ -170,13 +170,16 @@ class Canvas : public RefCountedDartWrappable<Canvas> {
 
   static void RegisterNatives(tonic::DartLibraryNatives* natives);
 
+  size_t ImageAllocationSize() { return image_allocation_size_; }
```

Comment:
Nit on naming convention (though this is a WebKit originating TU so the style is all over the place). `image_allocation_size` if inline and `GetImageAllocationSize` if OOL.

Reply:
Swtiching to `image_allocation_size`. I think this is right?

Reply:
Yeah.